packages/siop-oid4vp/lib/__tests__/TestUtils.ts

```diff
@@ -3,6 +3,7 @@
 import crypto, { createHash } from 'crypto'
 
 import { JwtPayload, parseJWT, SigningAlgo, uuidv4 } from '@sphereon/oid4vc-common'
+import { PartialSdJwtDecodedVerifiableCredential } from '@sphereon/pex/dist/main/lib'
```

Comment:
Import path issue (was already present by the looks of it)

Reply:
The import doesn't work for me from '@sphereon/pex'.

Reply:
That explains it being there I guess ;). Will fix it in PEX